### ISSUE TITLE
[Feature]: Support for other plot kwargs

### ISSUE THREAD

**Issue by etienneschalk** [etienneschalk/scientific-data-viewer#132]
### Feature Category

Data Visualization

### Priority

Low - Nice to have

### Feature Description

In #127 we can now add kwargs to the xarray plot call.

Other kwargs can be added to help visualization : 

```
vmin
vmax
add_colorbar
```

### Problem Statement

Sometimes the colorscale is not readable and we want to choose more carefully the vmin and vmax. Exemple 

<img width="444" height="382" alt="Image" src="https://github.com/user-attachments/assets/1132c671-8169-4c3e-b84a-7418a3091927" />

_Note: I attempted the use of `robust` kwargs, but did not change the plot ; potentially a bug where the robust kwargs is not taken into account_ ?

### Proposed Solution

Add other kwargs : vmin, vmax, add_colorbar.

Look for other kwargs too.

### Use Cases

- Working with data with outliers

### Alternatives Considered

Using the `robust` kwarg (already accessible via the plotting UI), without success ⚠️ 

### Code of Conduct

- [x] I have searched existing issues and this is not a duplicate
- [x] I have read and agree to follow the [Contributing Guidelines](CONTRIBUTING.md)

**Comment by etienneschalk** [comment]
Implemented in v0.11.0

**Comment by etienneschalk** [comment]
Bug #134

**Comment by etienneschalk** [comment]
Reopen because the goal is to support all plotting methods. See the design doc
